src/ontology/maxo-edit.owl

```diff
@@ -10755,6 +10761,45 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynony
 AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/MAXO_0010203> "echocardiogram procedure")
 SubClassOf(<http://purl.obolibrary.org/obo/MAXO_0010203> <http://purl.obolibrary.org/obo/MAXO_0000493>)
 
+# Class: <http://purl.obolibrary.org/obo/MAXO_0025001> (cardiac surgery)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "NCIT:C157806"^^xsd:string) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/MAXO_0025001> "A surgical procedure on the heart."^^xsd:string)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/MAXO_0025001> "heart surgery"^^xsd:string)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/MAXO_0025001> "cardiac surgery"^^xsd:string)
+SubClassOf(<http://purl.obolibrary.org/obo/MAXO_0025001> <http://purl.obolibrary.org/obo/MAXO_0000004>)
```

Comment:
This isn't wrong, but there is also the term 'surgical repair'. Maybe need to come up with a logical def so that heart repair would be inferred under both.

Reply:
I agree, good point. I can work on this.

Reply:
I created a PR: https://github.com/monarch-initiative/MAxO/pull/334